.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,9 @@
+- id: update-build-requirements
```

Comment:
Can we mention pre-commit hook in readme instead?

Reply:
You mean instead of creating this file?

Reply:
yes, or maybe I need explanation how this file will help to integrate hook to other repos

Reply:
+1 for the docs update, do we want to keep the actual pre-commit hook here? Cachito doesn't use those, nor does it declare build dependencies

Reply:
@chmeliik Do you know if its possible to add a hook from a third party repo?
I tried that without success...

(but maybe we could document to also copy the script - its overkill to install the whole cachito just to use a script that only relies on python standard library :thinking:)

Reply:
As for pre-commit dependency: one repo can contain pre-commit hooks w/o using them (and hence not relying on pre-commit).